app/src/main/res/values-de/strings.xml

```diff
@@ -130,6 +130,11 @@
     <string name="dialog_date_info_title_course">Kursdauer</string>
     <string name="dialog_date_info_title_section">Startdatum</string>
 
+    <string name="dialog_external_content_title">Externer Inhalt</string>
+    <string name="dialog_external_content_message">Diese Aufgabe wird in einem externen Tool geöffnet, welches nicht Teil der openHPI App ist. Dieses ist möglicherweise nicht für Mobilgeräte optimiert, weshalb die Bearbeitung an einem Computer empfohlen wird.\n\nSoll es dennoch geöffnet werden?</string>
```

Comment:
`openHPI` should be replaced with a dynamic flavor name.

Reply:
Done. Please check the German translation.